skll/experiments/utils.py

```
@@ -173,7 +175,7 @@ def _create_learner_result_dicts(
         # if there is no confusion matrix, then we must be dealing
         # with a regression model
         else:
-            learner_result_dict.update(result_dict)
+            learner_result_dict.update(result_dict)  # type: ignore
```

what is the ignored error here?